app/components/QuestionForProfilePage/index.js

```diff
@@ -147,11 +147,11 @@ export const QuestionForProfilePage = ({
   }
   return (
     <BaseStyled bordered={bordered && !isGeneral}>
-      {!isGeneral && (
+      {/*{!isGeneral && (
```

Comment:
please add TODO: comment

Reply:
i had problems when i tried to make push, so i solved to remove unused rows